mkdocs.yml

```diff
@@ -39,6 +39,7 @@ nav:
     - api-reference/index.md
     - api-reference/selectors.md
     - api-reference/typing.md
+    - api-reference/api-completeness.md
```

Comment:
Quite cool @FBruzzesi! ... like everything you do :-)
IMHO, shouldn't the `api-completeness.md` at the level of the other main pages? As a user, I would _probably_ not expect to see that here.. ?